Doc/c-api/function.rst

```diff
@@ -36,17 +36,8 @@ There are a few functions specific to Python functions.
 
    The function's docstring and name are retrieved from the code object. *__module__*
    is retrieved from *globals*. The argument defaults, annotations and closure are
-   set to ``NULL``. *__qualname__* is set to the same value as the function's name.
-
-
-.. c:function:: PyObject* PyFunction_NewWithQualName(PyObject *code, PyObject *globals, PyObject *qualname)
```

Comment:
Is it ok to remove this, or is it best to say that it was removed in 3.11?

Reply:
What do you mean with *this*? We cannot remove public c api functions and / or parameters or public c api functions per or backwards compatibility policy.

Reply:
Makes sense. I have restored it.